README.md

```diff
@@ -35,6 +35,17 @@ but if for example you see test errors on a version that's a full release later
 patches that will allow more recent versions to operate, ideally without
 requiring an upgrade to our current version.
 
+### Installing the easy way
+
+1) Run the setup script
+
```

Comment:
on mac, linux?

Reply:
Should I invest in targeting all platforms? To do this, I can either: 
- write in a cross-functional language (python, js, etc.)
- write an equivalent bat script

Reply:
We are explicitly supporting windows. Please note that the script is for Unix / Mac.

Reply:
Adding windows setup in #3396